stdlib/Test/src/logging.jl

```diff
@@ -4,7 +4,20 @@ using Logging: Logging, AbstractLogger, LogLevel, Info, with_logger
 import Base: occursin
 
 #-------------------------------------------------------------------------------
-# Log records
+"""
+    LogRecord
+
+Stores the results of a single logging call. Fields:
```

Comment:
```suggestion
Stores the results of a single log event. Fields:
```

Also in the list below, I think "log message" or "log event" is better than "call".  ("the LogLevel of the message" etc).